tft.py

```diff
@@ -279,17 +280,23 @@ def start_match() -> None:
 
 def shared_draft_pathing() -> None:
     """Navigate counter-clockwise in a diamond to help ensure a champ is picked up."""
-    auto.moveTo(946, 315)
-    click_right()
+    top = calculate_window_click_offset(window_title=CONSTANTS["window_titles"]["game"], position_x=946, position_y=315)
+    left = calculate_window_click_offset(
+        window_title=CONSTANTS["window_titles"]["game"], position_x=700, position_y=450
+    )
+    bottom = calculate_window_click_offset(
+        window_title=CONSTANTS["window_titles"]["game"], position_x=950, position_y=675
+    )
+    right = calculate_window_click_offset(
+        window_title=CONSTANTS["window_titles"]["game"], position_x=1200, position_y=460
+    )

```

Comment:
I'm realizing now, since we're doing this, we could probably switch these to just be % of screens and then it'd be more code that's moving away from needing to be hard-coded for 1920x1080 -- that said, definitely don't do it this PR, just food for future thought

Reply:
Yup, good shout :) Been thinking about doing that for the augment """selection""" already, but I think it's a lot more work to support other resolutions, so didn't do it yet.